leaflet.restoreview.js

```diff
@@ -1,31 +1,49 @@
-var RestoreViewMixin = {
-    restoreView: function () {
-        var storage = window.localStorage || {};
-        if (!this.__initRestore) {
-            this.on('moveend', function (e) {
-                if (!this._loaded)
-                    return;  // Never access map bounds if view is not set.
+(function() {
+    var RestoreViewMixin = {
+        restoreView: function () {
+            if (storageAvailable('localStorage')) {
```

Comment:
Instead of enclosing the whole block of code within this `if`, why not a quick return if not available?